libraries/AP_HAL_ChibiOS/shared_dma.h

```diff
@@ -103,4 +106,11 @@ class ChibiOS::Shared_DMA
         // point to object that holds the allocation, if allocated
         Shared_DMA *obj;
     } locks[SHARED_DMA_MAX_STREAM_ID+1];
+
+    // contention statistics
```

Comment:
A bit worried about the ram cost on F4. Maybe allocate this on first request to the file?

Reply:
This is static so in flash

Reply:
I have removed the timestamp, but the rest seems ok?